packages/visual-editor/src/store/tree.ts

```diff
@@ -109,9 +105,11 @@ export const useTreeStore = create<TreeStore>((set, get) => ({
      * rerendering the entire tree.
      */
     const treeDiff = getTreeDiffs({ ...currentTree.root }, { ...tree.root }, currentTree);
+    const didBreakpointsChange = hasBreakpointDiffs(currentTree, tree);
```

Comment:
Should we instead feed the breakpoints array? Not sure if we need to pass the entire trees 😓

Reply:
Saving a few lines within a quite large method was driving my code style, even though it's not nice to give the method access to the whole tree. Somehow, I didn't like to write the following:
```
const didBreakpointsChange = hasBreakpointDiffs(
  currentTree?.root?.data?.breakpoints ?? [],
  tree?.root?.data?.breakpoints ?? []
);
```
Do you have something else in mind, especially as we're only using lodash now within the method?